netbox_librenms_plugin/views/imports/actions.py

```
@@ -61,6 +60,57 @@ def _save_device(device) -> HttpResponse | None:
+    def _is_truthy(val):
+        return val.lower() in _TRUTHY if val is not None else False
+
+    _use_sysname_post = next((request.POST.get(k) for k in _USE_SYSNAME_KEYS if k in request.POST), None)
+    _use_sysname_get = next((request.GET.get(k) for k in _USE_SYSNAME_KEYS if k in request.GET), None)
+
+    if _use_sysname_post is not None:
+        use_sysname = _is_truthy(_use_sysname_post)
+    elif _use_sysname_get is not None:
+        use_sysname = _is_truthy(_use_sysname_get)
+    else:
+        pref = get_user_pref(request, "plugins.netbox_librenms_plugin.use_sysname")
```

Flake8 (7.3.0)

[error] undefined name 'get_user_pref'

(F821)